config/application.rb

```diff
@@ -60,6 +60,13 @@ class Application < Rails::Application
     # Version of your assets, change this if you want to expire all your assets
     config.assets.version = '1.0'
 
+    # Set cache headers
+    config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=31536000' }
+
+    # Errors raised within `after_rollback`/`after_commit` propagate normally
+    # like in other Active Record callbacks.
+    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
This is deprecated in Rails 5. I had removed this line in #1130, so I'm guessing it was re-added because of a conflict during a rebase

Reply:
http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#error-handling-in-transaction-callbacks ?

Reply:
That's applicable to Rails 4.2. In Rails 5.0 those exceptions are not suppressed and that line only shows a depreciation warning https://github.com/rails/rails/commit/07d3d402341e81ada0214f2cb2be1da69eadfe72